### PR TITLE
docs(install): fix build path error;

### PR DESCRIPTION
**Pull request by Rory-Z**

### CODE REVIEW

**Comment by hjianbo** [comment]
Plz amend the commit log

File: en_US/getting-started/install.md [hunk 1]
```diff
@@ -622,7 +622,7 @@ $ cd emqx-rel && make
 4. Start EMQ X Broker
 
 ```bash
-$ cd _build/emqx-rel/_rel/emqx
+$ cd _build/emqx/rel/emqx
 
 $ ./bin/emqx start
 emqx 4.0.0 is started successfully!
```
Comment (zmstone) [inline en_US/getting-started/install.md:628]: we should probably use a 4.3-beta.1 version as an example here

Comment (zmstone) [reply en_US/getting-started/install.md:628]: the same at line 632.

Comment (Rory-Z) [reply en_US/getting-started/install.md:628]: This is the 4.2 version of the document, so the previous compilation method will be used, and the 4.3 document will be updated later

**Review by zmstone — CHANGES_REQUESTED** [review]
@Rory-Z 
Please rebase it on release-4.2 branch
develop branch is deprecated.

**Comment by zmstone** [comment]
and another PR for `release-4.3` ?

**Pull request closed without merging**